src/tests/util/CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ add_library(ngraph_test_util STATIC EXCLUDE_FROM_ALL ${UTIL_SRC})
 
 ie_faster_build(ngraph_test_util UNITY)
 
-target_link_libraries(ngraph_test_util PUBLIC openvino::runtime openvino::core::dev openvino::util gtest gmock)
+target_link_libraries(ngraph_test_util PUBLIC openvino::runtime openvino::core::dev openvino::util gtest gmock
+    PRIVATE ov_shape_inference)
```

Comment:
As I understand, this change is to use `ov::SeqGen` in `set_shape_labels` helper.
Should such helepers like `ov::SeqGen` be moved to more common lib in the future?

Reply:
Could be moved to core `dev_api` will do next PR or if this will not merge into 2022.3.